### PR TITLE
Bump ProseMirror deps

### PR DESCRIPTION
**Pull request by kachkaev**
## 🌟 What is the purpose of this PR?

This PR is an extract from https://github.com/hashintel/hash/pull/201. It reduces PR scope and improves `git blame`. `git bisect` will be also easier if new ProseMirror deps bring us new bugs on their own.

## 🐾 Next steps

- Continue #201

## 🔍 What does this change?

- We have latest ProseMirror deps, which gives us the latest upstream features and may also help us with some bugfixing. All runtime changes are minor, according to semver.

## ❓ How to test this?

1.  Check CI

### CODE REVIEW

**Comment by kachkaev** [comment]
@nathggns marking as draft – there are some issues with typings

```

> @hashintel/hash-frontend
$ tsc --noEmit
Error: src/blocks/page/createEditorView.ts(64,13): error TS7006: Parameter 'currentNode' implicitly has an 'any' type.
Error: src/blocks/page/createEditorView.ts(64,26): error TS7006: Parameter 'currentView' implicitly has an 'any' type.
Error: src/blocks/page/createEditorView.ts(64,39): error TS7006: Parameter 'getPos' implicitly has an 'any' type.
Error: src/blocks/page/createEditorView.ts(76,15): error TS7006: Parameter 'currentNode' implicitly has an 'any' type.
Error: src/blocks/page/createEditorView.ts(76,28): error TS7006: Parameter 'currentView' implicitly has an 'any' type.
Error: src/blocks/page/createEditorView.ts(76,41): error TS7006: Parameter 'getPos' implicitly has an 'any' type.
Error: src/blocks/page/createEditorView.ts(98,16): error TS7006: Parameter 'node' implicitly has an 'any' type.
Error: src/blocks/page/createEditorView.ts(98,22): error TS7006: Parameter 'editorView' implicitly has an 'any' type.
Error: src/blocks/page/createEditorView.ts(98,34): error TS7006: Parameter 'getPos' implicitly has an 'any' type.
```

File: packages/hash/frontend/src/blocks/page/createEditorView.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ export const createEditorView = (
       );
     },
     nodeViews: {
-      block(currentNode, currentView, getPos) {
+      block(currentNode, currentView, getPos, _decorations) {
```
Comment (nathggns) [inline packages/hash/frontend/src/blocks/page/createEditorView.ts:64]: Might be worth adding a comment pointing to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57384#issuecomment-1018936089

Comment (kachkaev) [reply packages/hash/frontend/src/blocks/page/createEditorView.ts:66]: Yeah that’d be a useful reminder! I added a comment in each of the three cases.

**Review by nathggns — APPROVED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -14219,6 +14219,15 @@ prosemirror-commands@1.1.12:
...
     prosemirror-transform "^1.0.0"
 
+prosemirror-commands@1.2.1:
+  version "1.2.1"
+  resolved "https://registry.yarnpkg.com/prosemirror-commands/-/prosemirror-commands-1.2.1.tgz#eae0cb714df695260659b78ff5d201d3a037e50d"
+  integrity sha512-S/IkpXfpuLFsRynC2HQ5iYROUPiZskKS1+ClcWycGJvj4HMb/mVfeEkQrixYxgTl96EAh+RZQNWPC06GZXk5tQ==
+  dependencies:
+    prosemirror-model "^1.0.0"
+    prosemirror-state "^1.0.0"
+    prosemirror-transform "^1.0.0"
```
[the remark this answers is not in the document]
Comment (kachkaev) [reply yarn.lock]: Thanks a lot! Fixed.

**Pull request merged**